ex04.ex

```diff
@@ -55,7 +55,13 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min([ h | t ]), do: reduce([ h | t ], fn (h , t) ->
```

Comment:
# -1

Your `min` function does two things: it traverses the list  to return a value and it also finds the minimum of two numbers.  Each is a transformation, and therefore each is a function. I feel the minimum finding part should be split out.